### PR TITLE
Implement new design

### PR DESCRIPTION
**Pull request by jean**
Still needs some touches ... 

- logo in footer is blue-on-blue
- margins on inside pages (not frontpage) needs fixing

### CODE REVIEW

**Comment by bluenex** [comment]
@jean `.th` site isn't done yet? Also the footer seems to be stretched to be too longgg (Chrome only, see attached image).

image

**Comment by djay** [comment]
@bluenex more than the image streched. that footer isn't actually the footer and appearing on all pages. it needs to replace the other footer.

**Comment by bluenex** [comment]
@djay yeah it's not in `conf.py` yet. Need to work on it as well.

File: site/conf.py [hunk 1]
```diff
@@ -997,32 +997,37 @@
...
-    <p><a class="btn btn-primary btn-lg active" href="{sponsor_link}">{sponsor_link_text}</a>
-    </p>
+    <h1 class="text-white">{sponsors_heading}</h1>
+    <a class="btn-primary btn-lg get-ticket-btn text-sm block my-4 font-bold max-w-sm mx-auto" href="{sponsor_link}">{sponsor_link_text}</a>
   </div>
-  <div class="row">
-    <div class="col-md text-center">
-        <img src="/logo.png" class="footer-logo" alt="PyCon Thailand" />
+  <div class="row mx-4 my-4">
+    <div class="row">
```
Comment (bluenex) [inline site/conf.py]: Refer to this [SO thread](https://stackoverflow.com/a/34006377/4010864). This solves the stretching image issue on Chrome. We can also override `.row` which I'm not sure that will affect any other components of the site.

```suggestion
    <div class="row" style="align-items: flex-start;">
```

Comment (jean) [reply site/conf.py]: OK, applied

**Pull request merged**